lib/contracts/relationship.ts

```diff
@@ -0,0 +1,66 @@
+export const relationship = {
+	slug: `relationship`,
+	type: 'type@1.0.0',
+	name: 'Relationship',
+	data: {
+		schema: {
+			type: 'object',
+			required: ['name', 'slug', 'type', 'data'],
+			properties: {
+				name: {
+					type: 'string',
+				},
+				slug: {
+					type: 'string',
+					pattern: '^relationship-[a-z0-9-]+$',
+				},
+				type: {
+					type: 'string',
+					enum: ['relationship', 'relationship@1.0.0'],
```

Comment:
We can safely change this to 
```suggestion
					const: 'relationship@1.0.0',
```

Reply:
Done